websites/C/Coub/dist/metadata.json

```diff
@@ -1,23 +1,44 @@
 {
   "author": {
-    "name": "Mirax",
-    "id": "223355525826347018"
+    "name": "Cascade",
+    "id": "442612628104151040"
   },
+  "contributors": [
+    {
+      "name": "Mirax",
+      "id": "223355525826347018"
+    }
+  ],
   "service": "Coub",
   "description": {
     "ru": "Coub это сайт-видеохостинг, позволяющий своим пользователям публиковать короткие зацикленные видеоролики с несинхронным звуковым сопровождением — «коубы». ",
-    "en": "Coub is a video sharing website. It allows users to create and share looping videos up to ten seconds long, using existing video from YouTube, Vimeo, or their own files"
+    "en": "Coub is a video sharing website. It allows users to create and share looping videos up to ten seconds long, using existing video from YouTube, Vimeo, or their own files",
+    "de": "Coub ist eine Video-Sharing-Website, die sowohl für iOS als auch für Android verfügbar ist. Benutzer können Loop-Videos mit einer Länge von bis zu zehn Sekunden erstellen und freigeben, indem sie vorhandene Videos von YouTube, Vimeo oder ihren eigenen Dateien verwenden."
   },
   "url": "coub.com",
-  "version": "1.0.1",
-  "logo": "https://i.imgur.com/aZwjjLI.png",
-  "thumbnail": "https://i.imgur.com/fcUbUgZ.png",
+  "version": "2.0.0",
+  "logo": "https://i.imgur.com/HRAGMzw.png",
+  "thumbnail": "https://i.imgur.com/8P8RhcZ.png",
```

Comment:
maybe set the background of the logo transparent instead of that beige?

Reply:
its white: 
![image](https://user-images.githubusercontent.com/17952364/110528950-f6845d00-8118-11eb-8d88-b146e216aba0.png)

Reply:
huh, it is transparent, that is just chrome adding a background to the png.